pages/api/v1/bounty/curator/getAllCuratedBountiesAndChildBounties.ts

```
@@ -0,0 +1,173 @@
+
+interface Args {
+	page: number;
+	network: string;
+	curatorAddress: string;
+}
+
+export async function getAllBounties({ page, network, curatorAddress }: Args): Promise<IApiResponse<{ bounties: IBountyListing[]; totalBountiesCount: number }>> {
+	try {
+		if (!network || !isValidNetwork(network)) throw apiErrorWithStatusCode(messages.INVALID_NETWORK, 400);
+
+		if (isNaN(page) || !curatorAddress?.length) throw apiErrorWithStatusCode(messages.INVALID_PARAMS, 400);
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)